subiquitycore/models/network.py

```diff
@@ -360,6 +369,7 @@ def get_menu(self):
     # --- Model Methods ----
     def probe_network(self):
         log.debug('model calling prober.get_network()')
+        self.prober.probe()
```

Comment:
Why do we need to probe again? The probing should have already happened before, at the point where we did the initial discovery of the devices.